modules/tinymce/src/core/main/ts/focus/FocusController.ts

```diff
@@ -77,7 +77,7 @@ const registerEvents = (editorManager: EditorManager, e: { editor: Editor }) =>
     const element = SugarElement.fromDom(editor.iframeElement);
     const op = OuterPosition.find(element);
     const viewportBounds = WindowVisualViewport.getBounds(window);
-    if (op.top < viewportBounds.y || op.top > (viewportBounds.bottom - minimumVisibility)) {
+    if ((op.top + Height.get(element) - minimumVisibility) < viewportBounds.y || op.top > (viewportBounds.bottom - minimumVisibility)) {
```

Comment:
Why do we need the element height here? Isn't that checking the _bottom_ of the editor?

The test moves the editor completely off screen, so it will pass either way, but the changelog and PR title talk about the _top_ of the editor.

Reply:
Previously we checked the top of the editor against the top of the viewport, and if it was outside of it the editor was considered 'out of view' even if the editor already filled the screen ( which, if the editor is taller than the view, isn't that difficult ), and the scrolling occurred. This change moves it to check if the bottom of the editor is out of the top of the screen instead, meaning that it won't scroll if the editor is in view anymore. The already existing check checks if the top of the editor is lower than the view, so it's now properly contained.

Reply:
Looks correct to me the first one checks if the bottom of the editor is above the current viewport position. The second checks if the top is below the current viewport bottom position.
But unsure if we need the minimumVisibility not sure why that was added?

Reply:
But the PR title and ticket talk about whether the top of the editor is in view, and we didn't make any changes to checks comparing the editor's top position.

Sounds like just a changelog description issue, not a problem in the code.